src/redux/greetingsSlice.js

```diff
@@ -0,0 +1,37 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  messages: [],
+  status: 'idle',
+  error: null,
+};
+
+export const fetchGreetings = createAsyncThunk('messages/fetchGreetings', async () => {
+  const res = await fetch('http://localhost:3000/api/random_greeting');
```

Comment:
- [ ] Hi @Chaseknis, an awesome job so far :+1: I got the console error on the screenshot below when I tried to run the two apps:
![Screenshot from 2023-06-23 21-18-44](https://github.com/Chaseknis/hello_react_front-end/assets/63087491/39596019-2af5-438c-814f-0d44afd0113f)

- When attempting to make a crud request to the backend server(API) the error arises.
- Since we are expecting a request from any origin, let's kindly enable CORS in Rails API.

Reply:
Solved.

Reply:
> When attempting to make a crud request to the backend server(API) the error arises.
> 
> Since we are expecting a request from any origin, let's kindly enable CORS in Rails API.
> 

- [ ] 🤔 As requested by the pevious reviewer, This requierement should be addressed on the Backend repository. Kindly make this 👍🏻 You may check this article: [Rails CORS Guide: What It Is and How to Enable It](https://www.stackhawk.com/blog/rails-cors-guide/)

Reply:
- Well resolved, I see you have resolved the issue on the blackened 
<img width="480" alt="Screenshot 2023-06-24 at 15 05 08" src="https://github.com/Chaseknis/hello_react_front-end/assets/58692590/fa4637ac-f87c-422b-8f01-c7354b6265c0">

Reply:
- Thanks @Whoistolu, for checking this. 👍🏻